Makefile

```diff
@@ -1,5 +1,5 @@
 # Current Operator version
-VERSION ?= 4.8.0
+VERSION ?= 1.0-latest
```

Comment:
We may want to split the `VERSION` for the operator, which I would code to `1.0` at the moment, from the version of the _image_, which as @bpradipt suggested, we might want to set to `latest` for every build.

Reply:
see my other comment regarding _latest_. I think it only makes sense when everyone knows it's latest of master or latest of the 4.7 release etc. But if I'm the only one who thinks that way I'll be happy to change it :-)

Reply:
Hmm, thinking about it maybe it's best to use `latest` for the latest build of the latest release-x.y branch? and 4.7-latest, 4.6-latest for older release branches. Yes, I'll go with that unless there are other opinions?

Reply:
I'm going to leave VERSION in the Makefile at 1.0.0 because it's also used for the csv version not just the image tag. So to be clear the latest image should be tagged latest, but the version number of the operator and the ClusterServiceVersion is 1.0.0.